### PR TITLE
document fixes

### PR DESCRIPTION
**Pull request by rameshthoomu**
Fixed document issues
## Description

Added new targets in mkdocs.yaml file after PR#2150 merge. Also, added absolute paths in index.md and dev-setup documents.
## Motivation and Context

Fixes #
## How Has This Been Tested?

Tested this in readthedocs test account and observed build is successful. 
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff).
- [x] I have either added documentation to cover my changes or this change requires no new documentation.
- [x] I have either added unit tests to cover my changes or this change requires no new tests.
- [] I have run [golint](https://github.com/golang/lint) and have fixed valid warnings in code I have added or modified. This tool generates false positives so you may choose to ignore some warnings. The goal is clean, consistent, and readable code.

Signed-off-by:thoomu@us.ibm.com

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -11,17 +11,20 @@ pages:
 - Usecases: biz/usecases.md
 
 - Setup:
-  - Fabric Developer Setup: dev-setup/devenv.md
-  - Chaincode or Application Developer Setup: Setup/Chaincode-setup.md
-  - Java Chaincode Setup: Setup/JAVAchaincode.md
+  - Development Environment: dev-setup/devenv.md
```
Comment (christo4ferris) [inline mkdocs.yml]: Why did you rename this? I named it specifically to denote that this is the development environment for a Fabric developer. It really is not and should not be the default for application and chaincode developers.

File: mkdocs.yml [hunk 2]
```diff
@@ -11,17 +11,20 @@ pages:
...
+  - Building Fabric: dev-setup/build.md
+  - Chaincode Setup: Setup/Chaincode-setup.md
```
Comment (christo4ferris) [inline mkdocs.yml]: Why did you rename this?

**Comment by rameshthoomu** [comment]
@christo4ferris : updated mkdocs.yaml file with suggested changes.

**Comment by christo4ferris** [comment]
Thanks, LGTM - I'll merge if travis passes

**Pull request merged**